client/src/App.js

```diff
@@ -160,77 +173,68 @@ class Document extends React.Component {
   }
 }
 
-function DocumentFromRecipe({ document }) {
-  const sections = [];
+const renderers = {
```

Comment:
```suggestion
const RENDERERS = {
```

Reply:
This is done. I don't know if you think it should also move to the top alongside the OTHER CONSTANTS. (I think it should.)